src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -2092,5 +2095,15 @@ internal IOperation CreatePropertySubpatternMember(Symbol symbol, ITypeSymbol ma
                     return OperationFactory.CreateInvalidOperation(_semanticModel, nameSyntax, ImmutableArray<IOperation>.Empty, isImplicit);
             }
         }
+
+        private IInstanceReferenceOperation CreateLValuePlaceholderOperation(BoundLValuePlaceholder placeholder)
```

Comment:
>CreateLValuePlaceholderOperation [](start = 44, length = 32)

Do we have an IOperation test covering scenario with conversion?

Reply:
I don't see a test added for this.

---
In reply to: [325436181](https://github.com/dotnet/roslyn/pull/38732#discussion_r325436181) [](ancestors = 325436181)